--- /home/a12bbb/android-kernel/android_kernel_mt6765-main/drivers/misc/mediatek/connectivity/common/common_main/core/wmt_core.c	2026-08-30 18:46:25.000000000 +0800
+++ /home/a12bbb/kernel-patch-workspace/android_kernel_mt6765-main/drivers/misc/mediatek/connectivity/common/common_main/core/wmt_core.c	2026-09-14 23:54:56.426060878 +0800
@@ -33,9 +33,7 @@
 */
 #include <connectivity_build_in_adapter.h>
 #include "osal_typedef.h"
-#if 0
 #include "connsys_debug_utility.h"
-#endif
 #include "wmt_lib.h"
 #include "wmt_core.h"
 #include "wmt_ctrl.h"
