--- /home/a12bbb/android-kernel/android_kernel_mt6765-main/include/linux/cpumask.h	2026-08-30 18:46:25.000000000 +0800
+++ /home/a12bbb/kernel-patch-workspace/android_kernel_mt6765-main/include/linux/cpumask.h	2026-09-14 23:13:51.845291859 +0800
@@ -484,7 +484,9 @@
 static inline void cpumask_setall(struct cpumask *dstp)
 {
 	/* bitmap_fill() isn't optimized for compile-time constants */
-#if NR_CPUS <= BITS_PER_LONG
+#if NR_CPUS == BITS_PER_LONG
+	*cpumask_bits(dstp) = ULONG_MAX;
+#elif NR_CPUS < BITS_PER_LONG
 	*cpumask_bits(dstp) = BIT(NR_CPUS) - 1;
 #else
 	bitmap_fill(cpumask_bits(dstp), nr_cpumask_bits);
